src/test/resources/org/rust/ide/intentions/fixtures/unwrap_lambda_expr/unwrap_braces.rs

```diff
@@ -0,0 +1,3 @@
+fn main () {
+    |x| { x *<caret> x }
```

Comment:
@mkaput sorry, i don't get it 😂 , which part do you think is too invasive, i mean top-level lambda expression is illegal if it not wraped inside a function.